code/game/machinery/camera/camera.dm

```diff
@@ -395,6 +414,10 @@
 	assembly.update_icon()
 
 /obj/machinery/camera/portable/process() //Updates whenever the camera is moved.
-	if(GLOB.cameranet && get_turf(src) != prev_turf)
-		GLOB.cameranet.updatePortableCamera(src)
+	if(!part_of_camera_network)
+		return PROCESS_KILL // Stop wasting performance
+
+	if(part_of_camera_network && get_turf(src) != prev_turf)
+		SEND_SIGNAL(src, COMSIG_CAMERA_MOVED, prev_turf)
+		GLOB.cameranet.updatePortableCamera(src, prev_turf)
 		prev_turf = get_turf(src)
```

Comment:
```suggestion
	if(get_turf(src) == prev_turf)
		return
		
	SEND_SIGNAL(src, COMSIG_CAMERA_MOVED, prev_turf)
	GLOB.cameranet.updatePortableCamera(src, prev_turf)
	prev_turf = get_turf(src)
```
You already check for `part_of_camera_network` above, so we can just simplify this